src/main/scripts/releases/4.6/adhoc/tagFloristicsProtocols.js

```diff
@@ -0,0 +1,26 @@
+const floristicsDependentModules = [
```

Comment:
This script should be ...releases/4.8/ as it will need to be run as a part of the 4.8 release process.